kubectl-minio/static/crd.yaml

```diff
@@ -0,0 +1,1536 @@
+
```

Comment:
Why are we adding this file, seemes redundant to the Makefile target to generate it.